osu.Game/Screens/Play/MasterGameplayClockContainer.cs

```diff
@@ -43,7 +42,23 @@ public class MasterGameplayClockContainer : GameplayClockContainer
             Precision = 0.1,
         };
 
-        private double totalOffset => userOffsetClock.Offset + platformOffsetClock.Offset;
+        /// <summary>
+        /// An <see cref="ITrack"/> whose applied adjustments are considered as part of the gameplay itself, unrelated to the playback.
+        /// </summary>
+        public ITrack GameplayTrack { get; }
+
+        /// <summary>
+        /// The rate of gameplay when playback is at 100%.
+        /// This excludes any seeking / user adjustments.
+        /// </summary>
+        public double TrueGameplayRate => GameplayTrack.AggregateFrequency.Value * GameplayTrack.AggregateTempo.Value;
+
+        /// <summary>
+        /// The true gameplay rate combined with the <see cref="UserPlaybackRate"/> value.
+        /// </summary>
+        public double PlaybackRate => TrueGameplayRate * UserPlaybackRate.Value;
```

Comment:
what does this *mean* in human words? this xmldoc merely restates the code.

Reply:
Yeah my bad, I was planning to rewrite the xmldocs but forgot that while I was resolving the side issues pointed by the tests, I've explained enough now, feel free to reword or point out anything vague with it.